packages/experiments/src/components/Card/Card.types.ts

```diff
@@ -0,0 +1,44 @@
+import { IComponent, IComponentStyles, IHTMLSlot, IStyleableComponentProps } from '../../Foundation';
+import { IStackSlot } from '../../Stack';
+import { IBaseProps } from '../../Utilities';
+
+export type ICardComponent = IComponent<ICardProps, ICardTokens, ICardStyles>;
+
+// These types are redundant with ICardComponent but are needed until TS function return widening issue is resolved:
+// https://github.com/Microsoft/TypeScript/issues/241
+// For now, these helper types can be used to provide return type safety for tokens and styles functions.
+export type ICardTokenReturnType = ReturnType<Extract<ICardComponent['tokens'], Function>>;
+export type ICardStylesReturnType = ReturnType<Extract<ICardComponent['styles'], Function>>;
+
+export interface ICard {}
+
+export interface ICardSlots {
+  /**
+   * Defines root slot of the component.
+   */
+  root?: IHTMLSlot;
+
+  /**
+   * Defines a stack slot for managing the layout of the Card.
+   */
+  stack?: IStackSlot;
+}
+
+// Extending IStyleableComponentProps will automatically add stylable props for you, such as styles and theme.
+//    If you don't want these props to be included in your component, just remove this extension.
+export interface ICardProps extends ICardSlots, IStyleableComponentProps<ICardProps, ICardTokens, ICardStyles>, IBaseProps<ICard> {
+  /**
+   * Defines how to render the Card.
+   */
+  as?: string | React.ReactType<ICardProps>;
```

Comment:
It's not currently used here. I added it because I thought we might as well want to have it until we decide slots can completely take over the `as` prop functionality.

Reply:
Could we remove them for now and then, if we find we need them, analyze why can't Slots address the use case, then add back `as` prop only if needed?

Reply:
I don't have a strong opinion on this, so we can do it that way and refine as needed.

Reply:
OK let's remove for now. If we need to add it I'd like to figure out why and make sure it's not a shortcoming of Slots.